timescaledb/tutorials/grafana/grafana-timescalecloud.md

```diff
@@ -0,0 +1,66 @@
+# Connect TimescaleDB and Grafana
+Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
+source plugins that allow you to query and visualize data from a compatible
+database. To add a data source in Grafana, you must be signed in as a user with
+organization administration role privileges.
+
+To connect Grafana with a Timescale Cloud database, install Grafana. For
+information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+
+This section shows you how to connect a Timescale Cloud database as your data
+source in [Grafana][grafana-homepage].
+
+## Configure Timescale Cloud as a data source
+To configure a Timsecale Cloud database as the data source, first create a
+Timescale Cloud service. Then configure TimescaleDB as the data source in
+Grafana.
+
+<procedure>
+
+### Creating a Timescale Cloud service
+
+1.  Sign in to the [Timescale Cloud portal][tsc-portal].
+1.  Click `Create service`.
+1.  You can choose to build your service with or without demo data. 
+1.  Click `Download the cheatsheet`. This `.sql` file contains the credentials
+    that you need to configure TimescaleDB as a data source on Grafana.
+
+<highlight type="important">
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all our tutorials and run a
+few test projects of your own.
+</highlight>
```

Comment:
We can probably snippetize this

Reply:
Agreed!!!!!

Reply:
Hi, @Loquacity !!! Just created the partials.